DayrSuryan/DSEthiop10.xml

```diff
@@ -481,10 +488,23 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                </physDesc>
 
                <history>
-               <!--   <origin>
-                  </origin>-->
+                   <origin>
+                <origDate notBefore="1930" notAfter="1960">From 1930 to the mid-20th century.</origDate>
+              <persName ref="PRS5226haylase">King (<foreign xml:lang="gez">nǝguśǝna</foreign>) Ḫāyla Śǝllāse I</persName> is mentioned in <locus target="#30r"/>.
+                <note>The manuscript was copied and bound in Egypt. Several scribes participated in writing, unnamed <ref target="#h2"/> and <ref target="#h3"/>,
+                <persName ref="PRS12003HaylaM" role="scribe">Ḫāyla Māryām</persName> with <ref target="#h4"/>, <persName ref="PRS11943GabraS" role="scribe">Gabra Śǝllāse</persName> of <ref target="#h5"/>
+                 The commissioner and owner of the manuscript was Gabra Mikāʾel, most probably the same
+                  <persName ref="PRS12002Gabrami">ʾabuna Gabra Mikāʾel</persName> from Eritrea, associated with <placeName ref="INS0354DB">Dabra Bizan</placeName> and
+                <placeName ref="LOC2390DabraT">ʾƎndā ʾAbbonā</placeName>, who is meant in <ref target="#2"/>.
+                The name in Arabic imprinted on the spine, "al-qummuṣ Mīhāʾīl al-Ḥabašī", probably refers to
+                this Gabra Mikāʾel, just the first part of the name was left untranslated. Less likely,
```

Comment:
"leaving the first part of the name untranslated"?

Reply:
midified.